### ISSUE TITLE
No output with CLI, also wrong line numbers

### ISSUE THREAD

**Issue by itsimoshka** [sasstools/sass-lint#393]
I have `test.sass`:
``` sass
.splash
  bottom: 0
  left: 0
  margin: auto
  position: absolute
  top: 0
  right: 0
```

`.sass-lint`:

``` yml
files:
  include: 'test.sass'
options:
  formatter: html
  merge-default-rules: false
  output-file: 'linter/sass-lint.html'
rules:
  border-zero:
    - 1
    - convention: zero
  brace-style:
    - 1
    - allow-single-line: true
  clean-import-paths:
    - 1
    - filename-extension: false
      leading-underscore: false
  empty-line-between-blocks:
    - 1
  extends-before-declarations: 1
  extends-before-mixins: 1
  force-attribute-nesting: 1
  force-element-nesting: 1
  force-pseudo-nesting: 1
  function-name-format:
    - 1
    - allow-leading-underscore: true
      convention: hyphenatedlowercase
  hex-length:
    - 1
    - style: short
  hex-notation:
    - 1
    - style: lowercase
  indentation:
    - 1
    - size: 2
```

when I run

    sass-lint 

I have no output. But when I use 

    sass-lint  -v

I get output in `linter/sass-lint.html`
Also it contains:

```
5:1	Warning	Indentation of 0, expected 2	indentation
7:1	Warning	Indentation of 0, expected 2	indentation
9:1	Warning	Indentation of 0, expected 2	indentation
11:1	Warning	Indentation of 0, expected 2	indentation
13:1	Warning	Indentation of 0, expected 2	indentation
```

But why so strange line numbers?

**Comment by bgriffith** [comment]
Unfortunately he indentation rule is not yet working for that Sass syntax (only scss). We plan to add it when we adopt the new version of the parser we use.

I was under the impression that rules that don't apply or haven't yet been written for a syntax wouldn't run - @Snugug

**Comment by itsimoshka** [comment]
@bgriffith ok, I've removed `indentation` and added `property-sort-order` but still no out when I run

```
sass-lint
```

**Comment by itsimoshka** [comment]
@bgriffith And still problem with line numbers:

```
3:3	Warning	Expected `bottom`, found `left`	property-sort-order
5:3	Warning	Expected `left`, found `margin`	property-sort-order
7:3	Warning	Expected `margin`, found `position`	property-sort-order
9:3	Warning	Expected `position`, found `top`	property-sort-order
11:3	Warning	Expected `right`, found `bottom`	property-sort-order
13:3	Warning	Expected `top`, found `right`	property-sort-order
```

**Comment by DanPurdy** [comment]
Hi @itsimoshka you will only see errors or `severity: 2` output by the CLI if you don't include the verbose `-v` flag. This is the intended behaviour.

As for the line numbers, we'll have to look into it.

**Comment by itsimoshka** [comment]
@DanPurdy I expect results in file (in the example `linter/sass-lint.html`). but there is no result if I run without `-v` flag

**Comment by Snugug** [comment]
For 2.x we should think about how the CLI works to output results without `-v` and replace it with a `silent` flag of some sort. Until then @itsimoshka, outputting to a file is output, and without the `-v` flag, our CLI by design will not output anything.

**Comment by itsimoshka** [comment]
@Snugug in your CLI API docs:

```
Sass Lint can be run via its Command Line Interface (CLI). To do so, run sass-lint from the command line.
````
But in my case it doesn't run

**Comment by DanPurdy** [comment]
It does run, but to have the warnings show you need to use the -v(verbose output) flag, if you set a rule severity to 2 it will show you the errors with just `sass-lint`. The -q flag will list these errors rather than throw a single error. 

If you're seeing any problems then please post the error thrown.

**Comment by DanPurdy** [comment]
Closing this, it's working as intended but should be looked at as part of the 2.0 release #400
